### PR TITLE
TSUE-33 | @jdwjdwjdw | Adjust cards, lists, and hero styles

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- Top Banner updated styles, see [TSUE-26](https://stanfordits.atlassian.net/browse/TSUE-26): FE | Banners | Reduce height to 220 px
- [TSUE-33](https://stanfordits.atlassian.net/browse/TSUE-33): FE | Resource Banner image removal from List paragraph
- Accessibility fixups, see [TSUE-19](https://stanfordits.atlassian.net/browse/TSUE-19) Ally|Theme: Review for accessibility | homepage

# Review By (Date)
- ASAP

# Urgency
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Add top banners with example fields and confirm they match figma
image
3. Confirm the image is removed from the list paragraph type
4. Confirm [a11y issues](https://stanfordits.atlassian.net/browse/TSUE-19) have been addressed
5. Review code 🏒 

# Associated Issues and/or People
- [TSUE-19](https://stanfordits.atlassian.net/browse/TSUE-19) Ally|Theme: Review for accessibility | homepage
- [TSUE-26](https://stanfordits.atlassian.net/browse/TSUE-26): FE | Banners | Reduce height to 220 px
- [TSUE-33](https://stanfordits.atlassian.net/browse/TSUE-33): FE | Resource Banner image removal from List paragraph

[TSUE-26]: https://stanfordits.atlassian.net/browse/TSUE-26?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TSUE-33]: https://stanfordits.atlassian.net/browse/TSUE-33?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TSUE-19]: https://stanfordits.atlassian.net/browse/TSUE-19?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/scss/components/top-banner/_top-banner.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        }
+  
+        h2 {
+          font-size: 2.44140625em;
+          letter-spacing: -.016em;
+          font-weight: 900;
+          font-family: 'Source Serif Pro';
+  
+          @include grid-media-max('md') {
+            font-size: 2.0751953125em;
```
Comment (pookmish) [inline src/scss/components/top-banner/_top-banner.scss]: That's an interesting font size. can we just make it `2`?

File: src/scss/components/top-banner/_top-banner.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        @include grid-media-max('sm') {
+          width: 100%;
+        }
+
+        :first-child {
+          margin-top: 0;
+        }
+  
+        h2 {
+          font-size: 2.44140625em;
```
Comment (pookmish) [inline src/scss/components/top-banner/_top-banner.scss]: Let's round this to `2.5`

File: src/scss/components/top-banner/_top-banner.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          letter-spacing: -.016em;
```
Comment (pookmish) [inline src/scss/components/top-banner/_top-banner.scss]: why? this seems to be so small it's not worth the effort.

File: src/scss/components/card/_card.scss [hunk 4]
```diff
@@ -30,6 +22,15 @@
     }
   }
 
+  ul {
+    list-style: none;
+    padding-left: 0;
+
+    li {
+      margin-bottom: 0.1em;
```
Comment (pookmish) [inline src/scss/components/card/_card.scss]: this could probably just be `0` at that size.

**Review by pookmish — CHANGES_REQUESTED** [review]
I'll clone the site and review the results too. but here's a start

File: src/scss/components/top-banner/_top-banner.scss (hunk 3 above)
Comment (jdwjdwjdw) [reply src/scss/components/top-banner/_top-banner.scss]: Removed

File: src/scss/components/top-banner/_top-banner.scss (hunk 1 above)
Comment (jdwjdwjdw) [reply src/scss/components/top-banner/_top-banner.scss]: Sounds good, updated. I was referring to the similar SoE hero work, and those were the font-sizes being used there.

**Review by pookmish — APPROVED** [review]

**Pull request merged**